VagrantProvisionCentOS7.sh

```diff
@@ -94,6 +94,8 @@ sudo yum -y install \
     hoot-gdal-python \
     lcov \
     libicu-devel \
+    liboauthcpp \
+    liboauthcpp-devel \
```

Comment:
@bwitham and @mattjdnv any instances of hoot on bare metal will need to install these two RPMs from the Hoot-deps repo.  `sudo yum install -y liboauthcpp liboauthcpp-devel` will do the trick.